pyopia/io.py

```diff
@@ -180,6 +190,53 @@ def combine_stats_netcdf_files(path_to_data):
     return xstats, image_stats
 
 
+def merge_and_save_mfdataset(path_to_data, prefix='*'):
+    '''Combine a multi-file directory of STATS.nc files into a single '-STATS.nc' file
+    that can then be loaded with {func}`pyopia.io.load_stats`
+
+    Parameters
+    ----------
+    path_to_data : str
+        Folder name containing nc files with pattern '*Image-D*-STATS.nc'
+
+    prefix : str
+        Prefix to multi-file dataset (for replacing the wildcard in '*Image-D*-STATS.nc').
+        Defaults to '*'
+    '''
+
+    logging.info(f'combine stats netcdf files from {path_to_data}')
+    xstats, image_stats = combine_stats_netcdf_files(path_to_data, prefix=prefix)
+
+    settings = steps_from_xstats(xstats)
+
+    prefix_out = os.path.basename(settings['steps']['output']['output_datafile'])
+    output_name = os.path.join(path_to_data, prefix_out)
+
+    logging.info(f'writing {output_name}')
+    write_stats(xstats.to_dataframe(),
+                output_name,
+                settings,
+                image_stats=image_stats.to_dataframe())
```

Comment:
Consider supporting xr.Dataset for image_stats in write_stats to obviate need for to_dataframe here.